documentation/content/en/books/handbook/jails/_index.adoc

```diff
@@ -250,11 +250,15 @@ There are two ways to configure jails.
 The first one is to add an entry for each jail to the file [.filename]#/etc/jail.conf#.
 The other option is to create a file for each jail in the directory [.filename]#/etc/jail.conf.d/#.
 
-There is no right or wrong option.
-Each administrator must choose the one that best suits their needs.
-
 In case a host system has few jails, an entry for each jail can be added in the file [.filename]#/etc/jail.conf#.
-If the host system has many jails, it is good idea to have one configuration file for each jail in the [.filename]#/etc/jail.conf.d/# directory.
+If the host system has many jails, it is a good idea to have one configuration file for each jail in the [.filename]#/etc/jail.conf.d/# directory.
+
+The files in [.filename]#/etc/jail.conf.d/# must have `.conf` as their extension and have to be included in [.filename]#/etc/jail.conf#:
+
+[.programmlisting]
+....
+.include "/etc/jail.conf.d/*.conf";
+....
```

Comment:
I believe this phrase is outdated, especially considering FreeBSD versions 13 and 14.

See https://github.com/freebsd/freebsd-src/blob/main/libexec/rc/rc.d/jail#L122-L130

Perhaps consider changing it to something like this:
```
The files in [.filename]#/etc/jail.conf.d/# must be named as `jailname.conf`.
```

There is no need to use that include anymore.

Reply:
Not sure if I really understand you right. I double tested both, now. The files in `jail.conf.d` do have to have the extension (see screenshot) and if there is no /etc/jail.conf or just an empty one (without the .include statement) then jails configured in `jail.conf.d` will not autostart. (second screenshot). If I touch `/etc/jail.conf` the jail will not be started. So I do still believe that the extension and the include is needed. Just tested everything again to double check.

<img width="1010" alt="Bildschirmfoto 2024-02-26 um 09 15 16" src="https://github.com/freebsd/freebsd-doc/assets/45991777/c7304469-2d27-4258-ad25-c1776eb970c8">

<img width="525" alt="Bildschirmfoto 2024-02-26 um 09 18 40" src="https://github.com/freebsd/freebsd-doc/assets/45991777/e3d7ca5b-3299-4669-8275-361cfb41b82f">

Reply:
I see. I'd personally prefer the include because it appears to be more flexibel and less typing. I'd still like to update the handbook, because the way it is now jails simply don't start. Neither at system start, nor via `service jail start`. Shall I make it clear that there are two variants for the autostart? Or is the `.include` variant officially deprecated? I would ignore that, then.

Reply:
Thanks for your time reviewing this. Very much appreciated.